src/pages/DatasetCatalog.js

```diff
@@ -477,7 +473,7 @@ class DatasetCatalog extends Component {
                           td({ className: 'table-items cell-size ' + (!dataSet.active ? 'dataset-disabled' : '') }, [
                             a({
                               id: trIndex + '_linkTranslatedDul', name: 'link_translatedDul',
-                              onClick: this.openTranslatedDUL(dataSet.translatedUseRestriction),
+                              onClick: (e) => this.openTranslatedDUL(dataSet.dataUse),
```

Comment:
nit: unused click event
```suggestion
                              onClick: () => this.openTranslatedDUL(dataSet.dataUse),
```